zproject/urls.py

```diff
@@ -169,6 +169,14 @@
     url(r'^messages/flags$', 'zerver.lib.rest.rest_dispatch',
         {'POST': 'zerver.views.messages.update_message_flags'}),
 
+    # typing -> zerver.views.typing
+    # POST sends a typing notification event to a user
+    # POST to /typing/stopped sends a stopped typing notification event to a user
+    url(r'^typing$', 'zerver.lib.rest.rest_dispatch',
+        {'POST': 'zerver.views.typing.send_typing_notification_backend'}),
+    url(r'^typing/stopped$', 'zerver.lib.rest.rest_dispatch',
+        {'POST': 'zerver.views.typing.send_stopped_typing_notification_backend'}),
```

Comment:
Given how similar these routes are in terms of implementation, it's possible we should just have one endpoint and pass the `start/stop` as a POST parameter to it.  What do you think about that?